### PR TITLE
Change design for bbl and jumping arrow.

### PR DESCRIPTION
**Pull request by tiffanyb**
Make it less old-fashion.

### CODE REVIEW

File: angrmanagement/ui/widgets/qdisasm_graph.py [hunk 1]
```diff
@@ -540,15 +540,19 @@ def _draw_edges(self, painter, topleft_point, bottomright_point):
...
                 elif edge.sort == EdgeSort.TRUE_BRANCH:
                     # True branch
-                    color = Qt.darkGreen
+                    # Aqar
+                    color = QColor(0x79, 0xcc, 0xcd)
                 elif edge.sort == EdgeSort.FALSE_BRANCH:
                     # False branch
-                    color = Qt.red
+                    # Tomato
+                    color = QColor(0xf1,0x66,0x64)
```
Comment (ltfish) [inline angrmanagement/ui/widgets/qdisasm_graph.py]: Spaces after commas!

File: angrmanagement/ui/widgets/qblock.py [hunk 2]
```diff
@@ -115,16 +115,18 @@ def paint(self, painter):
         """
 
         # shadow
-        painter.setPen(QColor(0, 0, 0, 0))
-        painter.setBrush(QColor(0, 0, 0, 0x80))
-        painter.drawRect(self.x + 3, self.y + 3, self.width, self.height)
+        # painter.setPen(QColor(0, 0, 0, 0))
+        # painter.setBrush(QColor(0, 0, 0, 0x80))
+        # painter.drawRect(self.x + 3, self.y + 3, self.width, self.height)
```
Comment (ltfish) [inline angrmanagement/ui/widgets/qblock.py]: Just blow these lines away...

**Comment by ltfish** [comment]
Thanks! I took a look at the effects, and I like it much more than the previous one.

**Comment by tiffanyb** [comment]
Cleaned. :)

**Pull request merged**